numbers.rb

```diff
@@ -0,0 +1,26 @@
+def count(people, max)
+	reversed = false
+	person = 1
+	number = 1
+
+	while number <= max
+		inc = 1
+		skip = false
+
+		reversed = !reversed if number % 6 == 0
```

Comment:
I think the instructions say that you want 7 here. But the instructions are completely arbitrary and your logic looks good so: 👍